CONTRIBUTING.md

```diff
@@ -0,0 +1,117 @@
+# How to contribute
+
+👋 Hi! Great that you want to contribute to Intro-to-Serverless! 😃
+
+We're currently focussed on creating beginner content for Azure Functions written in Javascript, and using VSCode as the code editor.
+
+## Step 1 - GitHub Issues
+
+🔎 Please check if there's an [existing issue](https://github.com/bitprj/Intro-To-Serverless/issues) which matches your idea. Perhaps you can collaborate with someone on this.
+
+💡 If you have  new idea, please [create a new __Content Request__ issue](https://github.com/bitprj/Intro-To-Serverless/issues/new) first where you can describe the topic.
+
```

Comment:
A note about the best way is to help resolve an existing issue marked as good first issue.